build.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Denne filen må ha LF som line separator.
+ # Stop scriptet om en kommando feiler
+set -e
+ # Usage string
+usage="Script som bygger prosjektet
+ Bruk:
+./$(basename "$0") OPTIONS
+ Gyldige OPTIONS:
+    -h  | --help        - printer denne hjelpeteksten
+    --publish           - publiserer dockerimaget
+"
+ # Default verdier
+v=${versjon}
+IMAGE_NAME="personopplysninger"
+DOCKER_REGISTRY="docker.adeo.no:5000"
+DOCKER_REPOSITORY="personbruker"
+TAG="${DOCKER_REGISTRY}/${DOCKER_REPOSITORY}/${IMAGE_NAME}:${v:="unversioned"}"
+BUILDER_IMAGE="docker.adeo.no:5000/personbruker/personbruker-builder:1.1.0"
+ # Hent ut argumenter
+for arg in "$@"
+do
+case $arg in
+    -h|--help)
+        echo "$usage" >&2
+        exit 1
+        ;;
+    --publish)
+        PUBLISH=true
+        ;;
+    *) # ukjent argument
+        printf "Ukjent argument: %s\n" "$1" >&2
+        echo ""
+        echo "$usage" >&2
+        exit 1
+    ;;
+esac
+done
+
+ function build_command {
+    docker run \
+        --rm \
+        --volume $(pwd):/workspace \
+        --volume /var/run/docker.sock:/var/run/docker.sock \
+        --env NPM_TOKEN=${NPM_AUTH} \
+        --workdir /workspace \
+        $BUILDER_IMAGE \
+        "$@"
+}
+ function install_packages {
+    git status
+    ls -la
+    pwd
+    yarn
+}
+ function build_frontend {
```

Comment:
den kan slettes også for at docker build kjører alt vi trenger:
Dockerfile -> `RUN npm install && npm run build`